src/main/java/com/teamscale/upload/utils/OkHttpUtils.java

```diff
@@ -67,7 +67,9 @@ public static OkHttpClient createClient(boolean validateSsl, String trustStorePa
 		setTimeouts(builder, timeoutInSeconds);
 		builder.followRedirects(false).followSslRedirects(false);
 
-		configureTrustStore(builder, trustStorePath, trustStorePassword);
+		if (validateSsl) {
+			configureTrustStore(builder, trustStorePath, trustStorePassword);
+		}
 		if (!validateSsl) {
```

Comment:
I would prefer the following:

```java
if (!validateSsl) {
  disableSslValidation(builder);
} else {
  configureTrustStore(builder, trustStorePath, trustStorePassword);
}
```

Reply:
Generally, I would prefer it if you could format `disableSslValidation` and `configureTrustStore` a bit more similarly, so that it is clear that these are alternatives to each other.

Something like:

```java
List<TrustManager> trustManagers = new ArrayList<>();
trustManagers.addAll(getExternalTrustManagers(trustStorePath, trustStorePassword));
trustManagers.addAll(getJVMTrustManagers());
trustManagers.addAll(getOSTrustManagers());
MultiTrustManager multiTrustManager = new MultiTrustManager(trustManagers);
SSLContext sslContext = SSLContext.getInstance(PROTOCOL);
sslContext.init(null, new TrustManager[]{multiTrustManager}, new SecureRandom());
builder.sslSocketFactory(sslContext.getSocketFactory(), multiTrustManager);
```

and

```java
SSLContext sslContext = SSLContext.getInstance(PROTOCOL);
sslContext.init(null, new TrustManager[] { TrustAllCertificatesManager.INSTANCE }, new SecureRandom());
builder.sslSocketFactory(sslContext.getSocketFactory(), TrustAllCertificatesManager.INSTANCE);
builder.hostnameVerifier((hostName, session) -> true);
```

would make this more obvious.